dataproc/CHANGELOG.md

```diff
@@ -4,6 +4,22 @@
 
 [1]: https://pypi.org/project/google-cloud-dataproc/#history
 
+## 0.6.1
+
+11-12-2019 08:24 PST
+
+- docs: add python 2 sunset banner to documentation ([#9036](https://github.com/googleapis/google-cloud-python/pull/9036))
```

Comment:
Each changelog entry needs to be moved to the correct section below (`### Documentation` in this case).

Also, please strip the prefixes, capitalize, and add a period.  In the future, those changes will be applied by `release-please`, but it won't work until we split up the repository.

```suggestion
- Add python 2 sunset banner to documentation. ([#9036](https://github.com/googleapis/google-cloud-python/pull/9036))
```